bench/src/main/scala/sjsonnet/MainBenchmark.scala

```diff
@@ -40,26 +43,32 @@ object MainBenchmark {
 }
 
 @BenchmarkMode(Array(Mode.AverageTime))
-@Fork(4)
+@Fork(1)
 @Threads(1)
-@Warmup(iterations = 30)
-@Measurement(iterations = 40)
+@Warmup(iterations = 5)
+@Measurement(iterations = 10)
 @OutputTimeUnit(TimeUnit.MILLISECONDS)
 @State(Scope.Benchmark)
 class MainBenchmark {
 
   val dummyOut = MainBenchmark.createDummyOut
 
+  val cache = new DefaultParseCache
+
   @Benchmark
   def main(bh: Blackhole): Unit = {
-    bh.consume(SjsonnetMain.main0(
-      MainBenchmark.mainArgs,
-      new DefaultParseCache,
-      System.in,
-      dummyOut,
-      System.err,
-      os.pwd,
-      None
-    ))
+    bh.consume {
+      SjsonnetMain.main0(
+        MainBenchmark.mainArgs,
+        cache,
+        System.in,
+        dummyOut,
+        System.err,
+        os.pwd,
+        None
+      )
+      println("hi")
+      Thread.sleep(100000000)
```

Comment:
Was this file committed by accident? The changes should probably be excluded (this one in particular). If we're really changing what the benchmark runs, the README should be updated to reflect this.

Reply:
Yeah sorry, this branch turned into an experiement branch after I submitted it to. you guys for review. Instead: https://github.com/databricks/sjsonnet/pull/197 is the final PR